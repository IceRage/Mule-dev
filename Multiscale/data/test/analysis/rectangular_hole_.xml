<?xml version="1.0" encoding="utf-8"?>
<!--Warning! This xml file was automatically generated by a C++ program using the Boost PropertyTree library.-->
<experiment>
	<timepoint>
		<spatialEntity>
			<pseudo3D type="region">
				<clusteredness>0.9485033931976055</clusteredness>
				<density>0.5507625588592183</density>
				<area>177851.5</area>
				<perimeter>3163.024443149567</perimeter>
				<distanceFromOrigin>104.651803615609</distanceFromOrigin>
				<angle>97.0204545133727</angle>
				<shape>triangular</shape>
				<triangleMeasure>0.6177620728634406</triangleMeasure>
				<rectangleMeasure>0.5505283913624882</rectangleMeasure>
				<circleMeasure>0.4703064671806125</circleMeasure>
				<centroid>
					<x>292.8142</x>
					<y>682.5479</y>
				</centroid>
			</pseudo3D>
		</spatialEntity>
		<numericStateVariable>
			<name>avgClusterednessRegions</name>
			<value>0</value>
		</numericStateVariable>
		<numericStateVariable>
			<name>avgDensityRegions</name>
			<value>0.5507625588592183</value>
		</numericStateVariable>
	</timepoint>
</experiment>
