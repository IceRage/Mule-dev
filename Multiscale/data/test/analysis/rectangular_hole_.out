Clusteredness degree,Density,Area,Perimeter,Distance from origin,Angle(degrees),Shape,Triangle measure,Rectangle measure,Circle measure,Centre (x-coord),Centre (y-coord)
0.948503,0.550763,177852,3163.02,104.652,97.0205,triangular,0.617762,0.550528,0.470306,292.814,682.548

Average clusteredness degree: 0
Average density: 0.550763
